ragna/deploy/_cli/core.py

```diff
@@ -136,7 +136,10 @@ def wait_for_api() -> None:
                 while not check_api_available():
                     time.sleep(0.5)
 
-            wait_for_api()
+            wait_for_api(
```

Comment:
`wait_for_api` doesn't take any parameters. You have to pass them to the `@timeout_after` decorator.

Reply:
duh, sorry